app/models/mets_document/mods_document.rb

```diff
@@ -29,6 +29,30 @@ def title
           title
         end
       end
+      non_sort = value_from xpath: "mods:titleInfo/mods:nonSort"
+      return "#{non_sort} #{title_value}".gsub(/["\[\]]/, "") unless non_sort.empty?
+      title_value
+    end
+
+    def sort_title
+      normalize_whitespace(value_from(xpath: "mods:titleInfo/mods:title")).join(",")
+    end
+
+    def publisher
+      values = value_from(xpath: "mods:originInfo/mods:publisher")
+      values.join(" - ")
+    end
+
+    def date_other
```

Comment:
I think you should keep this `date_published`

Reply:
@tpendragon, Why do we want to display date_published with an array of: date_other, date_copyright, date_issued and also display the date_other, date_copyright, date_issued? Isn't this redundant?

Reply:
No, I'm saying I think `date_other` IS `date_published` - we don't want an array.

Reply:
oh ok, I didn't understand that date_other IS date_published. thanks